### ISSUE TITLE
Frame does not take picture correctly

### ISSUE THREAD

**Issue by TheWhite147** [janicduplessis/FibonacciClock#2]
Quand une photo est prise, elle ne correspond pas à ce qui est affiché dans le cadre bleu.

**Comment by FrancisGroleau** [comment]
Oui!! C'est que le rectangle bleu est en fonction de la dimension de ton écran et le crop ( le preview ) on prend la meme chose que le rectangle bleu pour cropper l'image, or la résolutio  de l'image n'est pas la même résolution que celle de l'écran!!
